docs/user-guide/sso.md

```diff
@@ -17,6 +17,7 @@ The following CLI flags are available for OIDC configuration:
 | `--external-auth-name` (Optional)       | Mapping of OIDC user information to username, e.g., `preferred_username`. By default, the `email` claim is used if available.                                        |
 | `--external-auth-full-name` (Optional)  | Mapping of OIDC user information to user's full name. Multiple elements can be combined, e.g., `name` or `firstName+lastName`. By default, the `name` claim is used. |
 | `--external-auth-avatar-url` (Optional) | Mapping of OIDC user information to user's avatar URL. By default, the `picture` claim is used if available.                                                         |
+| `--external-auth-default-active` (Optional) | True if new users should be automatically active. Default is false                                                                                               |
```

Comment:
On a second thought, I think new users should be active by default for backward compatibility. How about changing this to `--external-auth-default-inactive`?